### ISSUE TITLE
UI - Targets becomes blank in Global DNS Entries view up on existing from an Edit GDNS entry view which corrects itself on refresh.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#18977]
Rancher server version - Latest build from master - Mar 16

Steps to reproduce the problem:
Create a GDNS entry - GDNS1 that points to a MCAPP .
Create a GDNS entry  GDNS2 that points to a target project.

<img width="1165" alt="Screen Shot 2019-03-16 at 6 35 17 PM" src="https://user-images.githubusercontent.com/4266958/54483976-470e9b80-481a-11e9-9759-ce40c2e8cb75.png">

Edit GDNS1 and do no changes and "Cancel"
Edit GDNS2 and do no changes and "Cancel"

Notice that when you are brought to the ` Global DNS Entries` list page , target is empty for GDNS1.
<img width="1143" alt="Screen Shot 2019-03-16 at 6 31 54 PM" src="https://user-images.githubusercontent.com/4266958/54483958-e67f5e80-4819-11e9-922c-46e670e219e6.png">

I have also noticed targets becoming empty for GDNS2 as well.
<img width="1159" alt="Screen Shot 2019-03-16 at 6 31 23 PM" src="https://user-images.githubusercontent.com/4266958/54483961-ebdca900-4819-11e9-8b64-1e7b290e4c87.png">

These issues get corrected on refresh.

**Comment by davidnuzik** [comment]
Replicated on master HA setup with v2.2.10 UI.
Tested on latest master 3/18/19 022651b0ec13 and also replicated.

I follow the steps to reproduce. When I get to the step where I edit the first entry and cancel out I consistently see missing target for 2nd entry. I have reproduced several times.

![image](https://user-images.githubusercontent.com/45179589/54567331-ae4e5c00-4990-11e9-91d0-ca57882996fc.png)
